hooks/image/54-ek-launchers

```diff
@@ -0,0 +1,12 @@
+# Expose Endless Key launchers in the app grid (once a specific content pack has been downloaded)
+
+if [[ ! "${EIB_FLATPAK_REMOTE_FLATHUB_APPS}" =~ .*"org.endlessos.Key".* ]]; then
```

Comment:
This is nice but I think not necessary, it's not an error for a path in XDG_DATA_DIRS not to exist.

Reply:
(The conditionality is that only the image where this is wanted will list this hook.)